prober/kubeconfig.go

```diff
@@ -7,9 +7,10 @@ import (
 	"os"
 	"path/filepath"
 
+	"github.com/go-kit/log"
 	"github.com/prometheus/client_golang/prometheus"
 	"github.com/ribbybibby/ssl_exporter/config"
-	"gopkg.in/yaml.v2"
+	"gopkg.in/yaml.v3"
```

Comment:
There was a change related to `yaml.v2` in `go.mod` after I ran go mod tidy so I figured I would update it.